components/eam/src/physics/cam/hetfrz_classnuc_cam.F90

```diff
@@ -743,68 +762,68 @@ subroutine hetfrz_classnuc_cam_calc( &
    numice10s_imm_dst(:ncol,:) = 0._r8
    numice10s_imm_bc(:ncol,:)  = 0._r8
 
-   do i = 1, ncol
-      do k = top_lev, pver
+   do kk = top_lev, pver
+      do icol = 1, ncol
 
-         if (t(i,k) > 235.15_r8 .and. t(i,k) < 269.15_r8) then
-            qcic = min(qc(i,k)/lcldm(i,k), 5.e-3_r8)
-            ncic = max(nc(i,k)/lcldm(i,k), 0._r8)
+         if (t(icol,kk) > 235.15_r8 .and. t(icol,kk) < 269.15_r8) then
```

Comment:
there is a single-digit variable `t` left here. may need to rename it.

Reply:
I have replaced t with temperature. This part is in hetfrz_classnuc_cam_calc. I was going to refactor it after the PRs for get_aer_num and hetfrz_class_calc are merged.